.travis.yml

```diff
@@ -6,20 +6,13 @@ import:
 env:
   global:
     - REQUIRE_RECIPE="4.8.x-dev"
-    - REQUIRE_FRAMEWORKTEST="0.4.0"
+    - REQUIRE_FRAMEWORKTEST="0.4.2"
 
 # Using a manual jobs matrix in addition to jobs-fixed because there are:
 # - two different behat suites
 # - a manual release of storybook
 jobs:
   include:
-    - php: 7.1
-      env:
-        - DB=MYSQL
-        - REQUIRE_INSTALLER="$REQUIRE_RECIPE"
-        - NPM_TEST=1
```

Comment:
Doesn't look like any of those builds are running the JS tests. This is pretty bad.